infra/app-sync/README.md

```diff
@@ -0,0 +1,77 @@
+# AppSync GraphQL API
+
+This project sets up a GraphQL endpoint using **AWS AppSync**.
+
+Use the `lims` setup as a reference to create new endpoints based on other RDS data sources.
+
+---
+
+## Steps to Create a New AppSync GraphQL API
+
+### 1. Duplicate the Existing Configuration
+
+1. Copy the `lims` folder and rename it for your new API (e.g., `myapi`).
+2. Update the contents of `rds-data-config.json` to match the configuration for the new data source.
+
+---
+
+### 2. Generate the GraphQL Schema
+
+Use the `generate_schema.py` script to introspect the model and output a GraphQL SDL schema.
```

Comment:
Perhaps add a quick note why this is needed, e.g. that AppSync introspection does not directly output GraphQL SDL, which is needed to set up the graphql API endpoint.